test/components/converters/__init__.py

```diff
@@ -0,0 +1,3 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
I don't think we need `__init__.py` in test modules. Could you please explain if I am wrong?

Reply:
Ah, complex topic. OK to keep this change for the moment.